actions/class.RestQtiTests.php

```diff
@@ -136,9 +136,19 @@ public function import()
         }
     }
 
-    protected function getItemClassUri(): ?string
+    /**
+     * @throws common_exception_RestApi
+     */
+    protected function getItemClassUri(): string
```

Comment:
What about the getTestClassUri?

Reply:
It already contains an exception if class does not exists

Reply:
https://github.com/oat-sa/tao-core/blob/master/actions/class.RestController.php#L119